cob_hardware_config/cob4-4/config/diagnostics_analyzers.yaml

```diff
@@ -53,7 +53,7 @@ analyzers:
       base:
         type: diagnostic_aggregator/GenericAnalyzer
         path: Base
-        contains: 'base'
+        startswith: '/base'
```

Comment:
Fixed the base_drive_chain driver. Now all bases produce only one `/`. PR can be found here:  ipa320/cob_driver#244